news/views.py

```diff
@@ -526,6 +526,15 @@ def newsletters(request):
     })
 
 
+@require_safe
+@cache_page(30)
+def healthz(request):
+    """for use with healthchecks. wrapped with `cache_page` to test redis cache."""
+    # here to make sure DB is working
+    Newsletter.objects.exists()
```

Comment:
I thought about that. I guess it would be an issue if there were no newsletters in the DB. Could happen normally in theory, but in practice I think something would be very wrong. I'll add.